src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -75,12 +79,19 @@ public long amount() {
         throw new UnsupportedOperationException("amount() not yet implemented");
     }
 
-    // @todo #15:30min Implement prefix() by parsing the string representation
-    //  of transaction according to the pattern, described in the white
-    //  paper. Replace relevant test case with actual tests.
     @Override
     public String prefix() {
-        throw new UnsupportedOperationException("prefix() not yet implemented");
+        return new UncheckedText(
```

Comment:
@paulodamaso let's add a couple of validations here:

1. Prefix field actually exists (i.e. 4th element of the transaction line)
2. Prefix confirms to white paper spec (8-32 symbols, base64 characters)